Fix/spanish to english naming

luut04 opened the pull request:
Resumen de cambios realizados en el frontend:
✅ Interfaces renombradas:

[Coleccion](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [Collection](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
[Materiales](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [MaterialProduct](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
[Pedido](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [Order](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
[PresupuestoDetalle](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [BudgetDetail](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
PedidoDetalle → [OrderDetail](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
UpdatePedidoDto → [UpdateOrderDto](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
PresupuestoDetalleResponseDto → [BudgetDetailResponseDto](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
✅ Funciones renombradas:

loadPresupuestoToForm → [loadBudgetToForm](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
calculateTotalCosto → [calculateTotalCost](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
calculateCostosIndirectos → [calculateIndirectCosts](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
calculateGanancias → [calculateProfits](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
formatNumeroPresupuesto → [formatBudgetNumber](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
EditarMaterialPage → [EditMaterialPage](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
CrearMaterialPage → [CreateMaterialPage](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
✅ Variables locales renombradas:

[categoriaSlug](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [categorySlug](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
[categoria](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [category](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
✅ Rutas de API actualizadas:

[/clientes](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [/clients](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
[/presupuestos](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [/budgets](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
[/pedidos](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → /orders
[/productos](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → /products
[/materiales](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → [/materials](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
/categorias → [/categories](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html)
[/colecciones](vscode-file://vscode-app/c:/Users/luana/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) → /collections
/gastos-negocio → /business-expenses
/impuesto-general → /general-taxes

ecc97 committed:
fix: update upload materials select

elianisdev committed:
fix:Remove all `console.log` and `console.error` logs from the frontend code used during development.

    These logs were used solely for debugging and are no longer needed in the current version of the application.
    Removing them keeps the browser console clean, improves code quality and readability, and ensures a more professional frontend performance during delivery and demonstrations, without affecting the application's logic or behavior.

kevirui committed:
chore: remove console logs, errors and comments, it will be deploy

elianisdev committed:
fix:Remove all `console.log` and `console.error` logs from the frontend code used during development.

    These logs were used solely for debugging and are no longer needed in the current version of the application.
    Removing them keeps the browser console clean, improves code quality and readability, and ensures a more professional frontend performance during delivery and demonstrations, without affecting the application's logic or behavior.

elianisdev committed:
fix:Remove all `console.log` and `console.error` logs from the frontend code used during development.

    These logs were used solely for debugging and are no longer needed in the current version of the application.
    Removing them keeps the browser console clean, improves code quality and readability, and ensures a more professional frontend performance during delivery and demonstrations, without affecting the application's logic or behavior.

elianisdev committed:
fix:Remove all `console.log` and `console.error` logs from the frontend code used during development.

    These logs were used solely for debugging and are no longer needed in the current version of the application.
    Removing them keeps the browser console clean, improves code quality and readability, and ensures a more professional frontend performance during delivery and demonstrations, without affecting the application's logic or behavior.

luut04 committed:
feat: translate all Spanish naming to English - folders, functions, variables, interfaces, types, and API routes

luut04 committed:
feat: translate all Spanish naming to English - folders, functions, variables, interfaces, types, and API routes

XCODE89 reviewed (commented):
En un commit tan grande se dificulta la revisión, deberías hacer los cambios modulo por módulo, asi mantienes el control de lo que estas haciendo y sabes de que se trata.

elianisdev reviewed (commented):
Gracias por el aporte 🙌

Como sugerencia para próximos PRs, podría ayudar mucho separar los cambios en pasos más pequeños.

Veo que además del renombrado a inglés se generó un nuevo componente (probablemente sugerido por la IA).

Para facilitar la revisión, sería ideal que este tipo de cambios se hagan de forma incremental:

1. Primero el PR con el renombrado
2. Luego, si es necesario, un PR separado con la creación o refactor del componente

Así el review es más claro y controlamos mejor los cambios que introduce la IA.

Como recomendación

La IA suele ser un buen apoyo para avanzar más rápido, y a la vez es importante que entre todos revisemos bien los cambios que propone, ya que puede introducir ajustes extra sin que lo notemos y hacer cambios en el code no deseados. Cualquier cosa, si quieres, lo vemos juntas 😊

ecc97 committed:
fix: standardize routes to English naming convention in collections

ecc97 committed:
feat: Standardize routes to English naming convention

- Update navigation paths to use English route names
- Maintain Spanish folder structure for future migration
- Ensure consistent routing across all components

ecc97 committed:
Merge branch 'develop' into fix/modify-names-routes-pages

ecc97 committed:
🌐 Estandarización de Rutas a Convención en Inglés (#251)

<html>
<body>
<!--StartFragment--><h2>Descripción General </h2>
<p>
<img width="1395" height="214" alt="Captura de pantalla 2025-12-16
224804"
src="https://github.com/user-attachments/assets/67d0d802-ed02-4531-94aa-f173fdbfd5d9"
/>
</p>
<p>Este PR implementa la estandarización de todas las rutas de la
aplicación para usar convenciones de nombres en inglés, siguiendo parte
de la observación dada del feedback de foo y las buenas prácticas
modernas de desarrollo web orientadas a la internacionalización y la
mantenibilidad.</p>
<h2>Cambios Realizados</h2>
<h3>🔄 Estandarización de Rutas</h3>
<p>Se actualizaron todas las rutas de navegación a un esquema
consistente en inglés:</p>

Ruta en Español | Ruta en Inglés | Estado
-- | -- | --
/clientes | /clients | ✅ Actualizada
/colecciones | /collections | ✅ Actualizada
/materia-prima | /raw-materials | ✅ Actualizada
/presupuestos | /budgets | ✅ Actualizada
/pedidos | /orders | ✅ Actualizada
/gastos-negocio | /business-expenses | ✅ Actualizada


<h3>📁 Archivos Modificados</h3>
<ul>
<li>Componentes de navegación (<code>Sidebar.tsx</code>,
<code>NavigationTabs.tsx</code>)</li>
<li>Hooks con lógica de ruteo (<code>useCollections.ts</code>,
<code>useProducts.ts</code>, etc.)</li>
<li>Componentes de páginas con navegación interna</li>
<li>Configuración de ruteo en middleware</li>
</ul>
<h3>🎯 Beneficios Clave</h3>
<ul>
<li><strong>Consistencia</strong>: Todas las rutas siguen una convención
en inglés</li>
</ul>
<h3>📝 Notas</h3>
<ul>
<li>La estructura de carpetas en <code>/src/app/</code> se mantiene en
español por ahora (migración planificada a la espera en integración con
la otra PR que contiene las carpetas en inglés)</li>
<li>Todas las referencias internas a rutas fueron actualizadas a los
nuevos paths en inglés</li>
<li>Se mantiene compatibilidad con versiones anteriores mediante
redirecciones adecuadas</li>
</ul>
<h2>Pruebas</h2>
<ul>
<li>✅ Flujos de navegación verificados</li>
<li>✅ Acceso directo por URL validado</li>
<li>✅ Llamadas a <code>router.push()</code> actualizadas</li>
<li>✅ Atributos <code>href</code> de los enlaces estandarizados</li>
</ul>
<h2>Próximos Pasos</h2>
<p>Migrar la estructura de carpetas de español a inglés (a la espera de
la integración del otro PR relacionado con ello)</p>
<hr>
<h2>Adicionalmente hice una mejora en selección de materia prima (telas
e insumos)</h2>
<h3>¿Qué se hizo?</h3>
<ul>
<li>Se reemplazó el <code>CustomSelect</code> por un <strong>nuevo
componente <code>MaterialSearchSelect</code></strong>, que permite:
<ul>
<li>Buscar materiales por <strong>nombre, proveedor o
color</strong>.</li>
<li>Filtrar dinámicamente por tipo (<strong>tela</strong> o
<strong>insumo</strong>).</li>
<li>Visualizar información adicional del material (precio, proveedor,
colores).</li>
</ul>
</li>
<li>Se integró el nuevo selector tanto para <strong>telas</strong> como
para <strong>insumos</strong> dentro de
<code>RawMaterialTabForm</code>.</li>
</ul>
<h3>🛠 Corrección de comportamiento</h3>
<ul>
<li>Se corrigió el <strong>flasheo del componente
<code>EmptyMaterialsAlert</code></strong> al entrar en la pestaña de
<em>Materia Prima</em>.</li>
<li>Ahora el mensaje de “no hay materiales”:
<ul>
<li><strong>Solo se muestra cuando la carga finaliza
(<code>!isLoading</code>)</strong></li>
<li><strong>No hay error</strong></li>
<li><strong>Y realmente no existen materiales del tipo
correspondiente</strong></li>
</ul>
</li>
<li>Esto evita mostrar alertas incorrectas mientras la data aún se está
cargando.</li>
</ul>
<h3>🎯 Beneficio</h3>
<ul>
<li>Mejor experiencia de usuario al seleccionar materiales en listas
grandes.</li>
<li>Interfaz más clara, informativa y consistente.</li>
<li>Manejo correcto de estados de <strong>loading</strong> y
<strong>error</strong>.</li>
</ul>
<!-- notionvc: 0a958bc8-cb0c-4fee-8de8-0c82ceb97966
--><!--EndFragment-->
</body>
</html>

ecc97 committed:
Merge branch 'fix/spanish-to-english-naming' of https://github.com/FooTalentGroup/konfex-web-app into fix/spanish-to-english-naming

ecc97 commented:
Corregí algunos fixes y conflictos de esta rama que se generó con mis cambios, también tenía algunas variables e interfaces que no estaban nombradas correctamente que generaban errores con el build

Revisar también si puedes unos archivos duplicados que están de más, por ejemplo el de cliente profile es un duplicado de ya uno existente por el de ficha cliente

También algunas rutas de backend y service probarlas si funcionan, y demás cambios que no comprendo por la extensa cantidad de archivos cambiados, asegurarse también de ejecutar npm run build en el frontend antes de subir un cambio para ver si funciona en producción
